### PR TITLE
Start removing code from launcher.py.

### PR DESCRIPTION
**Pull request by oliverchang**
- Fix mutator plugins in minijail.
- Fix engine exit code checking.
- Move remaining tests into engine_test, and enable minijail tests.
- Remove libfuzzer_launcher_integration_test, and many tests from
  libfuzzer_launcher_test.

### CODE REVIEW

File: src/python/bot/fuzzers/libFuzzer/engine.py [hunk 1]
```diff
@@ -216,6 +223,13 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         artifact_prefix=reproducers_dir,
         extra_env=options.extra_env)
 
+    if (not environment.get_value('USE_MINIJAIL') and
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/libFuzzer/engine.py]: This means we don't get results for OSS-Fuzz, should try finding some workaround ?

File: src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py [hunk 2]
```diff
@@ -345,6 +390,19 @@ def setUp(self):
...
+
+@test_utils.integration
+class IntegrationTests(BaseIntegrationTest):
+  """Base libFuzzer launcher tests."""
+
+  def setUp(self):
+    BaseIntegrationTest.setUp(self)
+    self.crash_dir = TEMP_DIR
+
+  def compare_arguments(self, target_path, arguments, corpora, actual):
```
Comment (inferno-chromium) [inline src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py]: corpora_or_testcase ?

File: src/python/bot/fuzzers/libFuzzer/engine.py (hunk 1 above)
Comment (oliverchang) [reply src/python/bot/fuzzers/libFuzzer/engine.py]: I'm not sure what the best workaround here. Longer term we'll move from minijail to something more lightweight like chroot which should make this issue disappear.. but in any case this is a limitation that has always existed.

File: src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py (hunk 2 above)
Comment (oliverchang) [reply src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py]: Done!

**Pull request merged**